package.json

```diff
@@ -22,6 +22,7 @@
     "dist.build": "run-p build:icons build:js",
     "dist.clean": "del-cli dist/**/*.stories.tsx dist/design-tokens dist/**/*.css.map dist/*.*",
     "copy-to-dist": "copyfiles -u 1 src/fonts/* src/static/* src/icons/**/* dist",
+    "commit": "git cz",
```

Comment:
I might be doing something wrong, but after running `yarn install` this didn't work for me 🥲  also tried `yarn cz` but no dice. `./node_modules/.bin/cz` ended up working